lib/resources/example_json/r4_example_binary.rb

```diff
@@ -0,0 +1,13 @@
+module Cerner
+  module Resources
+    R4_BINARY_ENTRY = {
+      resourceType: "Binary",
```

Comment:
need to add double quotes around keys

Reply:
https://github.com/cerner/fhir.cerner.com/commit/70b171a1701f4afd9b85d8b10f6a8a609678eb5f#diff-9b4677563bd83e4f77cac2bb11986d7dR4